src/typings.d.ts

```diff
@@ -151,6 +151,13 @@ interface ClipParams {
    *  - See:      https://tonejs.github.io/docs/13.8.25/Volume
    */
   volume?: number;
+  /**
+   * Boolean parameter to trigger offline rendering. Use this to help
+   * If true, `scribbletune.clip` returns a `Tone.Player` with a buffer containing a pre-rendered sound of the sequence
+   * If alse, it returns a `Tone.Sequence` which does live rendering.
+   * Note: The rendering needs some time to complete & be saved in the buffer, before being able to playing.
```

Comment:
I m a bit curious about this aspect. I noted some discussion around this on Tone's issues section too. For current PR we are all good, but maybe we should consider some way to "announce" that the rendering is done.

We have a similar issue for the sample and samples props. In a presentation I gave, I actually kept the network tab open and waited for all the samples in my app to load completely before proceeding withe demo :P

Reply:
Yeah, I've had this concern too. I've thought about something like: allowing the user to pass a callback, which would be triggered at the end of the rendering. We may have 2 options to trigger this callback: after rendering of each clip or after rendering all the clips of the session/channel.

I didn't know what to chose for now (maybe we need to allow both ?), so I've simply put a `console.log` message at each beginning/end of rendering.

Reply:
I'll file an issue/feature request to keep track of this need for improvement.